extensions/amp-base-carousel/1.0/arrow.js

```diff
@@ -71,10 +63,88 @@ export function Arrow(props) {
  * @param {!JsonObject} props
  * @return {PreactDef.Renderable}
  */
-function renderDefaultArrow(props) {
+export function ArrowPrev(props) {
+  const {'customArrow': customArrow, ...rest} = props;
   return (
-    <button style={styles.defaultArrowButton}>
-      {props['dir'] < 0 ? '<<' : '>>'}
+    <Arrow
+      by={-1}
+      customArrow={customArrow || <DefaultArrow by={-1} />}
+      {...rest}
+    />
+  );
+}
+
+/**
+ * @param {!JsonObject} props
+ * @return {PreactDef.Renderable}
+ */
+export function ArrowNext(props) {
+  const {'customArrow': customArrow, ...rest} = props;
+  return (
+    <Arrow
+      by={1}
+      customArrow={customArrow || <DefaultArrow by={1} />}
+      {...rest}
+    />
+  );
+}
+
+/**
+ * @param {!JsonObject} props
+ * @return {PreactDef.Renderable}
+ */
+function DefaultArrow(props) {
+  const {by} = props;
+  const [hover, setHover] = useState(false);
+  const [active, setActive] = useState(false);
+  return (
+    <button
+      style={{
+        ...styles.defaultArrowButton,
+        stroke: hover ? '#222' : '#fff',
```

Comment:
Ideally we'd set `stoke="currentColor"` in SVG and we'd set `color: ...` in the styles.

Reply:
Done.